src/commonmark/en/content/sysadmin/audit.md

```diff
@@ -0,0 +1,48 @@
+# Audit
```

Comment:
I would add a simple description of what Auditing Scopes and Audit type.

The new Audit service makes use of two new concepts: Audit Scopes and Audit Type.

### Audit Scope

An Audit Scope is a logical area of  the application which can be audited. Currently there are three Audit Scopes:

- Tracker
- Metadata
- Aggregate

For the Tracker Audit Scope, the audited objects are: Tracked Entity Instance, Enrollment, Event, Tracked Entity Attribute Value.
For the Metadata Scope, all "metadata" objects are audited.
For the Aggregate Scope, the Aggregate Data Value objects are audited.

### Audit Type

An Audit Type is an action that triggers an audit operation. Currently we support the following types:

- READ
- CREATE
- UPDATE
- DELETE

As an example, when a new Tracked Entity Instance gets created, and if configured like so, the `CREATE` action is used to insert a new Audit entry in the `audit` db table.